package.json

```diff
@@ -83,7 +83,9 @@
     "rimraf": "^3.0.0",
     "ts-jest": "^25.5.0",
     "ts-node": "^8.10.1",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "amplify-cli-core": "^1.17.2",
```

Comment:
necessary for unit testing as otherwise, `peerDependencies` cannot be included using `require()` construct.